common/lib/xmodule/xmodule/tests/test_html_module.py

```diff
@@ -144,3 +144,24 @@ def test_index_dictionary_html_module_with_script_and_style_tags(self):
             "content": {"html_content": " This has HTML comment in it. HTML end. ", "display_name": "Text"},
             "content_type": "Text"
         })
+
```

Comment:
I verified the following test fails when run against unfixed master code:

'''
edxapp@precise64:~/edx-platform$ nosetests common/lib/xmodule/xmodule/tests/test_html_module.py:CourseInfoModuleTestCase --id-file /edx/app/edxapp/edx-platform/.testids/lms/noseids --xunit-file /edx/app/edxapp/edx-platform/reports/lms/nosetests.xml --verbosity=1 --with-xunit --xunit-file=/edx/app/edxapp/edx-platform/reports/lms/nosetests.xml
F
-----------------------------------------------------------------------------
1) FAIL: testUpdatesRender (xmodule.tests.test_html_module.CourseInfoModuleTestCase)

   Traceback (most recent call last):
    common/lib/xmodule/xmodule/tests/test_html_module.py line 167 in testUpdatesRender
      self.fail("CourseInfoModule could not parse an invalid date!")
   AssertionError: CourseInfoModule could not parse an invalid date!
'''